python/cudf/cudf/tests/test_array_ufunc.py

```diff
@@ -62,6 +63,43 @@ def test_ufunc_cudf_series_cupy_array(np_ar_tup, func):
 )
 def test_error_with_null_cudf_series(func):
     s_1 = cudf.Series([1, 2])
-    s_2 = cudf.Series([1, 2, None])
+    s_2 = cudf.Series([1, None])
+
+    # this thows a value error
+    # because of nulls in cudf.Series
     with pytest.raises(ValueError):
         func(s_1, s_2)
+
+    s_1 = cudf.Series([1, 2])
+    s_2 = cudf.Series([1, 2, None])
+
+    # this throws a type-error because
+    # indexes are not aligned
+    with pytest.raises(TypeError):
+        func(s_1, s_2)
+
+
+@pytest.mark.parametrize(
+    "func", [lambda x: np.absolute(x), lambda x: np.sign(x)],
```

Comment:
Since this is just directing the argument into a numpy function without modifying it, is a lambda needed or could the functions just be passed directly as params? Lambdas are a little harder to debug on the command line as well

Reply:
Agreed,  should be able to switch to a normal function.

Reply:
Addressed by https://github.com/rapidsai/cudf/pull/6839/commits/665234136db75eb1b1024e125bff853df6cf27a1